src/traces/waterfall/attributes.js

```diff
@@ -53,20 +62,20 @@ module.exports = {
 
     constraintext: barAttrs.constraintext,
 
-    cliponaxis: barAttrs.cliponaxis,
```

Comment:
Actually, having `cliponaxis` from the start would be nice. @archmoj are there any complications with `cliponaxis` that I can't think of?

Reply:
Added back. 
Not sure why on the mock below it continues clipping the first trace even if it was set to false. 
https://github.com/plotly/plotly.js/blob/fed3c97af3997a99e05ddb7fc18a5220d571c235/test/image/baselines/waterfall_cliponaxis-false.png

Reply:
Yeah, looks like `cliponaxis` isn't working.